ocs_ci/ocs/ui/views.py

```
@@ -2375,37 +2375,32 @@
+_ENCRYPTION_SUMMARY_ENABLED = (
+    "//dt[@data-test='detail-item-title' and normalize-space()='Encryption']"
+    "/following-sibling::dd[@data-test='detail-item-value']"
+    "//button[normalize-space()='Enabled' or normalize-space()='Not enabled')]",
+    By.XPATH,
+)
+_ENCRYPTION_SUMMARY_CLOSE = (
+    "//div[@role='dialog']//button[@aria-label='Close']",
+    By.XPATH,
+)
+_ENCRYPTION_SUMMARY_CONTENT = (
+    "//div[@role='dialog']" "[.//button[@aria-label='Close']]" "[.//h6]",
```

Ruff (0.15.20)

[warning] Implicitly concatenated string literals on one line

Combine string literals

(ISC001)